README.md

```diff
@@ -48,6 +48,8 @@ Do you need some help with Orchard Core? Don't worry, there are ways to get help
 - [X (Twitter)](https://twitter.com/orchardcms)
 - [LinkedIn](https://www.linkedin.com/groups/13605669/)
 - [Facebook](https://www.facebook.com/OrchardCore)
+- [LinkedIn](https://orchardcore.net/linkedin)
```

Comment:
This is already there above. But the other one can be removed because this is a better link.